### PR TITLE
Fixes issue with deleting a post while looking at it in data view.

### PR DESCRIPTION
**Pull request by ushahidlee**
- Added deleted post state to event bus
- Pick up delete event and republish through event bus.

### CODE REVIEW

File: apps/web-mzima-client/src/env.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "production": true,
-  "backend_url": "https://mzima-dev-api.staging.ush.zone/",
+  "backend_url": "http://localhost:8080/",
```
Comment (Angamanga) [inline apps/web-mzima-client/src/env.json:3]: Is this change intentional?

**Review by Angamanga — APPROVED** [review]
Works all fine! Just a question, is the change of backend_url intentional? It does make sense since that is the url for the API locally, right?

File: apps/web-mzima-client/src/env.json (hunk 1 above)
Comment (ushahidlee) [reply apps/web-mzima-client/src/env.json:3]: Good spot, that was an oversight!

Comment (Angamanga) [reply apps/web-mzima-client/src/env.json:3]: Although I think it does makes sense, that is the address the API get when setting it up locally, isn't it? It makes it easier for new contributors to get started.
